lightning/src/routing/router.rs

```diff
@@ -76,6 +76,9 @@ pub struct Route {
 	///
 	/// [`Event::PaymentPathFailed`]: crate::util::events::Event::PaymentPathFailed
 	pub payment_params: Option<PaymentParameters>,
+	/// List of penalties corresponding to the paths in the `paths` vector.
+	/// This is populated by taking the last [`PathBuildingHop`] of every path found in [`find_route`]
+	pub path_penalties: Vec<u64>,
```

Comment:
Hmm, do we care about actually exposing this programmatically? Presumably the user can just call their own scorer immediately after getting the route if they really want this info. I think just logging it may be sufficient.

Reply:
hi @TheBlueMatt, thanks for taking a look! that is a good suggestion -- I'll change my implementation. I will take another stab over the weekend :)